dspace-api/src/main/java/org/dspace/discovery/indexobject/factory/IndexObjectFactoryFactory.java

```diff
@@ -68,7 +68,7 @@ public IndexFactory getIndexableObjectFactory(String indexableObjectUniqueString
      */
     public IndexFactory getIndexFactoryByType(String indexableFactoryType) {
         for (IndexFactory indexableObjectFactory : getIndexFactories()) {
-            if (indexableObjectFactory.getType().equals(indexableFactoryType)) {
+            if (StringUtils.equalsIgnoreCase(indexableObjectFactory.getType(), (indexableFactoryType))) {
```

Comment:
The parentheses around `indexableFactoryType` can be removed on this line.

Reply:
Removed